src/lightbox/LightboxFooter.js

```diff
@@ -20,9 +20,11 @@ const styles = createStyleSheet({
     fontSize: 14,
     alignSelf: 'center',
   },
+  iconTouchTarget: {
+    alignSelf: 'center',
+  },
   icon: {
     fontSize: 28,
```

Comment:
detected. So, with a font size of 28, the new width and height of
    the pressable area are approx. 10 + 28 + 10 = 48.

It is kind of odd that we're specifying this with something called `fontSize`. There's not really a font involved here. And it makes me wonder if this size is getting scaled by the user's font-size setting -- which would be the wrong behavior, because that setting should only affect actual text that the user may be trying to read as text.

OK, a quick test finds that it isn't (as of v27.158, current beta), neither on Android nor iOS. If I crank the system font-size setting way up, the text in the header and footer get bigger (as they should), but the icons stay the same size (as they should.)

Still, it'd be good if we can avoid that confusion. And as your "approx." suggests, it's not entirely clear that `fontSize` of 28 means a size of 28px.

It looks like in other places where we create an icon, we tend to use its own `size` prop, like `size={24}`, instead of using the `style` prop. If we do that here (with the same number as for `fontSize`), does that keep the behavior unchanged?